Source/Core/Core/HW/WiimoteCommon/WiimoteReport.h

```diff
@@ -166,7 +166,7 @@ static_assert(sizeof(OutputReportSpeakerData) == 21, "Wrong size");
 // FYI: Also contains LSB of accel data:
 union ButtonData
 {
-  static constexpr u16 BUTTON_MASK = ~0x6060;
+  static constexpr u16 BUTTON_MASK = ~0x60e0;
```

Comment:
Just checking, this is meant to include everything but `acc_bits` , `unknown` and `acc_bits2` - right?

Reply:
Yes. The `unknown` was previously included, but not used, so it probably wasn't hurting anything.